packages/contracts-bedrock/package.json

```diff
@@ -16,19 +16,20 @@
   "scripts": {
     "build:forge": "forge build",
     "build:differential": "tsc scripts/differential-testing.ts --outDir dist --moduleResolution node --esModuleInterop",
+    "build:scripts": "(cd scripts/merkle-trie-fuzzer && go build main.go)",
```

Comment:
I think you don't need the `cd`, you should be able to do `go -o <path-to-binary-ouput> build scripts/merkle-trie-fuzzer`